src/components/entity/update.vue

```diff
@@ -169,9 +169,7 @@ const currentVersion = computed(() =>
     overflow-y: auto;
   }
 
-  .modal-body { padding: 0; }
```

Comment:
`EntityUpdateRow` uses `$padding-left-modal-header`. Now that the padding here isn't being zeroed out, maybe `EntityUpdateRow` doesn't need to use it anymore?

Reply:
I've changed some of the padding here so the update table looks nice.
<img width="925" alt="Screenshot 2025-04-23 at 4 26 51 PM" src="https://github.com/user-attachments/assets/bbdd6356-55a7-48b1-b70e-cc45df9b0c68" />